conf/test_experimental.config

```diff
@@ -29,10 +29,13 @@ params {
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.samplesheet.csv'
     matrix =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.gene_counts.top1000cov.tsv'
     contrasts =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.contrasts.csv'
-    pathway = "propd,propd_grea,propr,cor"
+
+    // analysis
+    pathway = "propd,propd_grea,propd_ora,propr,cor,deseq2_ora"
```

Comment:
propd_fdr is currently a bit useless for this test, because it is not giving any significant results for the test dataset, so it can be ignored.
But I would have to check other datasets for it to run and test

Then instead of pcorbshrink maybe I would ignore cor